src/measure_theory/bochner_integration.lean

```diff
@@ -2,11 +2,40 @@
 Copyright (c) 2019 Zhouhang Zhou. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Zhouhang Zhou
-
-Bochner integral on real normed space
 -/
 
 import measure_theory.simple_func_dense
+import analysis.normed_space.bounded_linear_maps
+
+/-!
+# Bochner integral
+
+To be added in the next PR.
+
+## Main definitions
+
```

Comment:
Can you add here a paragraph on the strategy to define the Bochner integral (define it on simple integrable functions and then extend it by uniform continuity), explain the notations you use for simple integrable functions, how these relate to simple functions, integrable functions, and so on, to help the reader?